.zsh-autosuggestionsrc

```diff
@@ -0,0 +1,13 @@
+# Setup zsh-autosuggestions
+source $HOME/.zsh-autosuggestions/autosuggestions.zsh
+
+# Enable autosuggestions automatically
+zle-line-init() {
+    zle autosuggest-start
+}
+
+zle -N zle-line-init
+
+# use ctrl+t to toggle autosuggestions(hopefully this wont be needed as
+# zsh-autosuggestions is designed to be unobtrusive)
+bindkey '^T' autosuggest-toggle
```

Comment:
Yeah, I’ve already omitted it from the basic instructions in [readme](https://github.com/tarruda/zsh-autosuggestions/blob/master/README.md); it’s mentioned only in the configuration section.